extension/js/common/settings.ts

```diff
@@ -369,6 +383,19 @@ export class Settings {
     await Ui.delay(1000);
   }
 
+  /**
+    * determines how to treat old values when changing account
+    */
+  private static getOverwriteMode = (key: string): 'fallback' | 'forget' | 'keep' => {
+    if (key.startsWith('google_token_') || ['uuid', 'rules', 'openid', 'full_name', 'picture', 'sendAs'].includes(key)) { // old value should be used if only a new value is missing
+      return 'fallback';
+    } else if (key.startsWith('passphrase_')) { // force forgetting older values
+      return 'forget';
+    } else { // keep old values if any
+      return 'keep'; // todo: should 'keys' be kept?
```

Comment:
not sure if we should keep 'keys' as there is a special section of code using `destAccountPrivateKeys` vartiable to topy them

Reply:
I'll leave it up to you. If we already handle keys with a special purpose code, then you can just use that.

Reply:
You're right - sounds like we should use `forget` for `keys`.